tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -1362,8 +1375,33 @@ private void handleFormatUnit(XPathParts parts, String unitPattern, List<String>
         if (parts.getElement(-1).equals("unitPattern")) {
             String count = parts.getAttributeValue(-1, "count");
             DecimalQuantity amount = getBest(Count.valueOf(count));
+
             if (amount != null) {
-                addFormattedUnits(examples, parts, unitPattern, shortUnitId, amount);
+                if (shortUnitId.equals("light-speed")) {
```

Comment:
Ideally this added code should go in a subroutine. It should be possible to view an entire function on the screen without scrolling.

Reply:
I tend to agree for very long functions. But this does fit on a page.